bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/handler/TibberHandler.java

```diff
@@ -0,0 +1,473 @@
+                }
+                if (myObject.has("maxPower")) {
+                    updateChannel(LIVE_MAXPOWER, myObject.get("maxPower").toString());
+                }
+                if (myObject.has("voltagePhase1")) {
+                    updateChannel(LIVE_VOLTAGE1, myObject.get("voltagePhase1").toString());
+                }
+                if (myObject.has("voltagePhase2")) {
+                    updateChannel(LIVE_VOLTAGE2, myObject.get("voltagePhase2").toString());
+                }
+                if (myObject.has("voltagePhase3")) {
+                    updateChannel(LIVE_VOLTAGE3, myObject.get("voltagePhase3").toString());
+                }
+                if (myObject.has("currentPhase1")) {
+                    updateChannel(LIVE_CURRENT1, myObject.get("currentPhase1").toString());
+                }
+                if (myObject.has("currentPhase2")) {
+                    updateChannel(LIVE_CURRENT2, myObject.get("currentPhase2").toString());
+                }
+                if (myObject.has("currentPhase3")) {
+                    updateChannel(LIVE_CURRENT3, myObject.get("currentPhase3").toString());
+                }
+                if (myObject.has("powerProduction")) {
+                    updateChannel(LIVE_POWERPRODUCTION, myObject.get("powerProduction").toString());
+                }
+                if (myObject.has("accumulatedProduction")) {
+                    updateChannel(LIVE_ACCUMULATEDPRODUCTION, myObject.get("accumulatedProduction").toString());
+                }
+                if (myObject.has("minPowerProduction")) {
+                    updateChannel(LIVE_MINPOWERPRODUCTION, myObject.get("minPowerProduction").toString());
+                }
+                if (myObject.has("maxPowerProduction")) {
+                    updateChannel(LIVE_MAXPOWERPRODUCTION, myObject.get("maxPowerProduction").toString());
+                }
+            } else {
+                logger.debug("Unknown live response from Tibber");
+            }
+        }
+
+        private void sendMessage(String message) throws IOException {
+            logger.debug("Send message: {}", message);
+            Session session = TibberHandler.this.session;
+            if (session != null) {
+                session.getRemote().sendString(message);
+            }
+        }
+
+        public void startSubscription() {
+            String query = "{\"id\":\"1\",\"type\":\"start\",\"payload\":{\"variables\":{},\"extensions\":{},\"operationName\":null,\"query\":\"subscription {\\n liveMeasurement(homeId:\\\""
+                    + myHome
+                    + "\\\") {\\n timestamp\\n power\\n lastMeterConsumption\\n accumulatedConsumption\\n accumulatedCost\\n currency\\n minPower\\n averagePower\\n maxPower\\n voltagePhase1\\n voltagePhase2\\n voltagePhase3\\n currentPhase1\\n currentPhase2\\n currentPhase3\\n powerProduction\\n accumulatedProduction\\n minPowerProduction\\n maxPowerProduction\\n }\\n }\\n\"}}";
```

Comment:
This string is way too long for this line, can you split it up?

Reply:
Corrected.